packages/protocol/contracts/tokenvault/BridgedERC1155.sol

```diff
@@ -94,21 +94,21 @@ contract BridgedERC1155 is EssentialContract, ERC1155Upgradeable {
         _mintBatch(_to, _tokenIds, _amounts, "");
     }
 
-    /// @dev Burns tokens.
+    /// @dev Batch burns tokens.
     /// @param _account Address from which tokens are burned.
-    /// @param _tokenId ID of the token to burn.
-    /// @param _amount Amount of tokens to burn.
-    function burn(
+    /// @param _ids Array of IDs of the tokens to burn.
+    /// @param _amounts Amount of tokens to burn respectively.
+    function burnBatch(
         address _account,
-        uint256 _tokenId,
-        uint256 _amount
+        uint256[] memory _ids,
+        uint256[] memory _amounts
```

Comment:
shall we change this function to external, then we can change 'memory' to 'calldata' to save some gas?